packages/space-opera/src/redux/poster_dispatcher.ts

```diff
@@ -0,0 +1,13 @@
+import {registerStateMutator, State} from './space_opera_base';
```

Comment:
rename this file to poster_dispatchers - plural.

Reply:
Done